bofire/benchmarks/LookUpTableBenchmark.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import pandas as pd
+
+from bofire.benchmarks.benchmark import Benchmark
+from bofire.data_models.domain.api import Domain
+
+
+class LookUpTableBenchmark(Benchmark):
+    """This class is a benchmark for discrete data where you can lookup a table for new inputs.
+    It takes in a file with inputs and outputs and returns output for matching inputs from that file.
+
+    Args:
+        Benchmark: Subclass of the Benchmark function class.
+    """
+
+    def __init__(
+        self,
+        domain: Domain,
+        LookUpTable: pd.DataFrame,
```

Comment:
```suggestion
        lookup_table: pd.DataFrame,
```

just call it `lookup_table`, arguments are always snake case. You also have to adjust below.